client/src/components/Recipe.tsx

```diff
@@ -1,10 +1,14 @@
 // Package imports
+import React from 'react';
 import { Card, Badge } from 'react-bootstrap';
 
 export default function Recipe ({ recipe }) {
+
+  // console.log(recipe);
+
   // Clean up ingredients string into unique array
-  const ingredientsArray = [...new Set(recipe.ingredients.split(','))];
-  let firstOne = ingredientsArray[0];
+  const ingredientsArray: Array<string> = [...new Set(recipe.ingredients.split(','))] as Array<string>;
```

Comment:
Does it make a difference if we use `Array<string>` instead of `string[]`? Cause I think the square bracket notation is TS's syntactic sugar on `Array<type>` and makes it look a bit cleaner.